metron-maven-archetypes/README.md

```diff
@@ -0,0 +1,56 @@
+# Metron Maven Archetypes
+
+These are [Apache Maven Archetypes](http://maven.apache.org/archetype/index.html) for use in creating extension components for the Apache Metron system
+Archetypes are helpful in creating maven based projects with the correct setup and dependency configurations for the target system, including providing sample implementations.
+
+### metron-maven-parser-extension-archetype
+This is an archetype for creating an Apache Metron Parser Extension
+
+#### Use
+
+Build and install the archetype
+```
+$ mvn install
+```
+
+Create a directory to host your extension code
+
+```
+$ mkdir ~/src/my-parser-extension
+$ cd ~/src/my-parser-extension
+```
+
+Use the archetype to create your project
+
+```
+$ mvn archetype:generate -DarchetypeCatalog=local
```

Comment:
I'm following through these docs to create a parser. I did the archetype install, but the metron archetype is not showing in my list of available archetypes. Did I miss a step? I'll look at maven archetype install docs, but if there's anything you can think of in the meantime it might be good to get it added to the doc.